cmd2/cmd2.py

```diff
@@ -3520,14 +3520,9 @@ def do_history(self, args: argparse.Namespace) -> Optional[bool]:
                     else:
                         fobj.write('{}\n'.format(command.raw))
             try:
-                # Handle potential edge case where the temp file needs to be quoted on the command line
-                quoted_fname = utils.quote_string(fname)
-
-                # noinspection PyTypeChecker
-                self.do_edit(quoted_fname)
-
+                self._run_editor(fname)
```

Comment:
Not directly calling the **edit** command see within the **history** command is a good thing as it removes the possibility of a pathological infinite loop which then allows us to put **edit** commands in the history.